### DISCUSSION TITLE
BenchmarkDotNet 0.13.12

### DISCUSSION THREAD

**Discussion by AndreyAkinshin** [dotnet/BenchmarkDotNet#2495] [Releases]
Full changelog: https://benchmarkdotnet.org/changelog/v0.13.12.html

## Highlights

The biggest highlight of this release if our new VSTest Adapter,  which allows to run benchmarks as unit tests in your favorite IDE!
The detailed guide can be found [here](https://benchmarkdotnet.org/articles/features/vstest.html
).

This release also includes to a minor bug fix that caused incorrect job id generation:  fixed job id generation ([#2491](https://github.com/dotnet/BenchmarkDotNet/pull/2491)).

Also, the target framework in the BenchmarkDotNet templates was bumped to .NET 8.0.

<hr /><em>This discussion was created from the release <a href='https://github.com/dotnet/BenchmarkDotNet/releases/tag/v0.13.12'>0.13.12</a>.</em>

**Comment by rchoffardet** [comment]
Hey @AndreyAkinshin, the link to the detailed guide is missing. Is it this one? https://benchmarkdotnet.org/articles/features/vstest.html

**Reply by AndreyAkinshin** [reply]
@rchoffardet yes, this is the link.

Thanks for the comment, I have fixed the link in the description. This announcement is generated automatically, and the highlights are obtained from the docs that use docfx link format (`xref:docs.vstest`). The release workflow requires an additional step of transforming such links. Will try to implement it.
